server/src/testFixtures/java/io/crate/testing/LogicalPlanAssert.java

```diff
@@ -35,10 +35,6 @@ public class LogicalPlanAssert extends AbstractAssert<LogicalPlanAssert, Logical
     @Nullable
     private final PlanStats planStats;
 
-    protected LogicalPlanAssert(LogicalPlan actual) {
```

Comment:
strange, I thought I checked it and it was complaining to override the ctor from superclass, apologies :(

Reply:
Afaik a class cannot force a parent class constructor to have the same signature. The only constraint is that one of the visible constructors must be used.